globalbibletools/platform
# feat: email verification

**What's Changed** 
--
Brought over the email verification logic from the old global bible tools

**QA**
--
1. Navigate to the profile view.
2. Update the email. Confirm that the email has not been updated in the database. Confirm that an email has been "sent" with a verification link.
3. Navigate to the verification link. Confirm that the user's email has been updated.
